docs/core/diagnostics/metrics-instrumentation.md

```diff
@@ -542,6 +547,66 @@ Name                                                  Current Value
 > [!NOTE]
 > OpenTelemetry refers to tags as 'attributes'. These are two different names for the same functionality.
 
+## Using Advice to customize Histogram instruments
+
+Consumers of Histogram instruments can choose different aggregation strategies.
+The default mode when using OpenTelemetry is [Explicit Bucket Histogram
+Aggregation](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#explicit-bucket-histogram-aggregation).
+
+When using explicit bucket aggregation the measurements recorded through a
+Histogram will be grouped into buckets which track the sum and count of values
+to that bucket.
+
+The default bucket configuration is described in the OpenTelemetry
+Specification: `[ 0, 5, 10, 25, 50, 75, 100, 250, 500, 750, 1000, 2500, 5000,
+7500, 10000 ]`.
+
+The default values may not lead to the best granularity for every Histogram. For
+example, sub-second request durations would all fall into the `0` bucket.
+
+To solve this problem the `9.0.0` version of the
+`System.Diagnostics.DiagnosticSource` package introduced the
+(<xref:System.Diagnostics.Metrics.InstrumentAdvice%2A>) API.
+
+The `InstrumentAdvice` API may be used by instrumentation authors to specify the
+set of recommended default bucket boundaries for a given Histogram. Consumers
+can then choose to use those values when configuring aggregation.
```

Comment:
could you add a note about -> more buckets gives more precise data, but at the cost of more memory/cpu, so users wont go and add millions of buckets.

Reply:
Added an "IMPORTANT" section to cover that